organizing_hub/templates/calls/callcampaign_form.html

```diff
@@ -67,6 +70,19 @@ <h3>Contact Info</h3>
 
         </div>
 
+        <h3>Callers</h3>
+        <p>
+          Add callers to your calling campaign. These users will be able to
+          login and make calls using the email address listed after creating an
+          account. You may add or remove callers after campaign creation.
```

Comment:
Ditto on language. Also if we're going to reuse content then please put into a central partial or message variable.